### PR TITLE
[docs] changes needed for mandel.cdt docs generation

### PR DESCRIPTION
**Pull request by iamveritas**
## Change Description
1. README.md and LICENSE.md filles are not needed
2. add the copy step which brings over all the docs content from mandel.cdt to local markdown folder
3. add a new shell script which is running all steps: delete, init, create folders, copy, generate
4.  add capi and core folders to be processed by doxygen

### CODE REVIEW

File: scripts/clean_gen_docs.sh [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+######
+# main
+# check for parameters
+if [ -z $ROOT_DIR ]; then
+  echo "missing '-d' directory, '-h' for help"; exit 1;
+fi
+
+echo "delete directory ${ROOT_DIR}"
+
+rm -rf "${ROOT_DIR}"
```
Comment (ericpassmore) [inline scripts/clean_gen_docs.sh:52]: Want to change this to 
rm -rf ${ROOT_DIR}/devdocs ${ROOT_DIR}/reference 

Removing entire ROOT_DIR is too destructive

File: scripts/clean_gen_docs.sh [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+# compute script dir for copying files from here to web directory
+SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+
+echo "change dir to ${SCRIPT_DIR}"
+
+pushd "${SCRIPT_DIR}"
+
+pwd
```
Comment (ericpassmore) [inline scripts/clean_gen_docs.sh:61]: confusing, should remove or add context 
printf "Currently working in: %s" $(pwd)

File: scripts/clean_gen_docs.sh [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+
+echo "run ./initialize_repository.sh -d ${ROOT_DIR}"
+./initialize_repository.sh -d "${ROOT_DIR}"
+
+echo "run ./run_me_to_gen_docs.sh -u -d "${ROOT_DIR}""
```
Comment (ericpassmore) [inline scripts/clean_gen_docs.sh:66]: double set of quotes : please remove quotes around ${ROOT_DIR}

File: scripts/clean_gen_docs.sh (hunk 1 above)
Comment (iamveritas) [reply scripts/clean_gen_docs.sh:52]: but that's what I want, a 100% clean start

**Review by ericpassmore — APPROVED** [review]

**Pull request merged**